internal/tiltfile/tiltfile.go

```diff
@@ -34,9 +34,6 @@ func Load(ctx context.Context, filename string, matching map[string]bool) (manif
 	tfRoot, _ := filepath.Split(absFilename)
 
 	s := newTiltfileState(ctx, absFilename, tfRoot)
-	defer func() {
```

Comment:
this doesn't seem to do anything, but maybe it has something to do with named returns that I'm not understanding?

Reply:
seems to work ok for me
https://play.golang.org/p/N2e7s_wsABh

can you give a specific example?

Reply:
if there's an error, i think this function needs to return non-nil? i can write a test for it to double-check the behavior if you want.

Reply:
Ohhh I think I get it now. Okay I'll add it back.